core/src/test/resources/HistoryServerExpectations/one_app_multi_attempt_json_expectation.json

```diff
@@ -11,6 +14,9 @@
     "completed" : true
   }, {
     "attemptId" : "1",
+    "startTimeEpoch" : 1426533910242,
+    "endTimeEpoch" : 1426533945177,
+    "lastUpdatedEpoch" : "",
```

Comment:
The empty string is actually hard-coded in HistorySeverSuite, I just copied how it was done with lastUpdated for lastUpdatedEpoch